flash/vision/backbones.py

```diff
@@ -15,10 +15,14 @@
 
 import torchvision
 from pytorch_lightning.utilities import _BOLTS_AVAILABLE, rank_zero_warn
+from pytorch_lightning.utilities import _module_available
 from pytorch_lightning.utilities.exceptions import MisconfigurationException
 from torch import nn as nn
 from torchvision.models.detection.backbone_utils import resnet_fpn_backbone
 
+if _module_available('timm'):
```

Comment:
Please, use _TIMM_AVAILABLE as for _BOLTS_AVAILABLE

Reply:
but _BOLTS_AVAILABLE comes from lightning. Should I define a new variable and then use it? @tchaton